### PR TITLE
Add Foreman 3.1 and Katello 4.3 release notes

### PR DESCRIPTION
**Pull request by ekohl**
This does not add the Katello 4.3 specific notes nor the Katello contributors, but at least it's a start.

### CODE REVIEW

File: guides/doc-Release_notes/topics/foreman.adoc [hunk 1]
```diff
@@ -4,8 +4,63 @@
...
+Users are encouraged to upgrade to Ubuntu 20.04 either on Foreman 2.5 or 3.0.
+
+Note that this is only support to run Foreman and Foreman Proxy themselves on Ubuntu 18.04. Managing Ubuntu 18.04 systems remains supported. See https://community.theforeman.org/t/deprecation-plans-for-foreman-on-el7-debian-10-and-ubuntu-18-04/25008[the RFC] for more information.
+
 [id="foreman-deprecations"]
 == Deprecations
+
+=== The `:unattended` Setting has been deprecated
+
+Historically Foreman has supported the `:unattended` setting in `settings.yaml`. This is barely tested and has seen various regressions because of that.
```
Comment (ekohl) [inline guides/doc-Release_notes/topics/foreman.adoc:55]: I added this sentence compared to the ones on the website. While writing https://github.com/theforeman/theforeman.org/pull/1939 I changed it slightly. I'm not sure what the best phrasing is, but we should align them all :)

**Comment by lzap** [comment]
Sure I am good, but linkchecker is failing and also web does not build. Can you rebase and fix it?

**Comment by ekohl** [comment]
> Sure I am good, but linkchecker is failing and also web does not build. Can you rebase and fix it?

It is based on the current 3.1 so is 3.1 just broken now?

**Comment by ekohl** [comment]
I opened https://github.com/theforeman/foreman-documentation/pull/963 and that does fix build-web but there check-html also fails. So this isn't further regressing.

**Comment by lzap** [comment]
Thanks for looking into this, the damn psych is everywhere. I merged the other PR.

**Comment by ekohl** [comment]
This is green now.

**Comment by lzap** [comment]
Thank you very much for your contribution, this is now merged.

**Pull request merged**